Topic/dockerfile update

chavafg opened the pull request:
This PR contains 2 commits that : 
- Remove old Dockerfiles that no longer worked because of changes in cc-oci-runtime
- Add new Dockerfile for building cc-oci-runtime in Ubuntu.

chavafg committed:
Add Dockerfile to build cc-oci-runtime in Ubuntu 16.04.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

sameo commented on cc2.1/build_cc/ubuntu/Dockerfile:
```diff
@@ -0,0 +1,71 @@
+#  This file is part of cc-oci-runtime.
```
Could this path please be `2.1/build/ubuntu/Dockerfile` ? the `cc` prefix and suffix is redundant, we know we're building Clear Containers :)

chavafg replied:
sure

sameo commented on cc2.1/build_cc/ubuntu/Dockerfile:
```diff
@@ -0,0 +1,71 @@
+
+# Create directory for clonning cc-oci-runtime
+RUN mkdir -p /root/go/src/github.com/01org
+
```
Instead of creating the directory and git cloning the code, could you please try to do a:

`go get -d github.com/01org/cc-oci-runtime`

chavafg replied:
tried that, but got this:

```
package github.com/01org/cc-oci-runtime: no buildable Go source files in /root/go/src/github.com/01org/cc-oci-runtime
The command '/bin/sh -c go get github.com/01org/cc-oci-runtime' returned a non-zero code: 1
```
and the docker build command cannot continue

mcastelino replied:
@chavafg can you try go get -d
This should not build it

dlespiau replied:
argh! go get -d will still try to compile the package. What it doesn't do is go install :/

sameo commented on clearlinux/Dockerfile:
```diff
@@ -1,50 +0,0 @@
-#
```
Can we please all the existing files and move them to `2.1/run/<distro>` instead ?
I know they're broken, but if we remove them we will have a hard time fixing them...

chavafg replied:
sounds good

chavafg committed:
Move current dockerfiles to 2.1/run directory

2.1/run directory was created to track files that belong
to 2.1 and its purpose is to run clear-containers

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

dlespiau commented on README.md:
```diff
@@ -1,5 +1,5 @@
-Those are Dockerfiles for building Ubuntu, Fedora and Clear Linux
-containers capable of running the Clear Container container runtime
-as the default OCI runtime.
+This repoository contains different Dockerfiles for building Ubuntu,
```
s/repoository/repository/

dlespiau commented on cc2.1/build_cc/ubuntu/Dockerfile:
```diff
@@ -0,0 +1,71 @@
+ENV HOME /root
+WORKDIR $HOME
+
+# Clone and clone cc-oci-runtime
```
clone and clone? :)
